### ISSUE TITLE
Julia code looks weird 

### ISSUE THREAD

**Issue by cormullion** [mossr/julia-tufte-beamer#3]
Out of curiosity, is the code typesetting in the example.pdf typical?

<img width="857" alt="Screenshot 2021-04-06 at 09 22 54" src="https://user-images.githubusercontent.com/52289/113681680-298b2380-96ba-11eb-95d2-300d396c8ada.png">

It looks a bit odd to me - I'm not sure if it's how LATEX/Beamer normally looks though. For presentations, there are some very small and faint symbols...

**Comment by mossr** [comment]
Yeah this is typical for this style. The subscripts come out teeny-tiny and the mix of serif font with a monospaced font feels odd. Note this _isn't_ using JuliaMono (yet)
